public/images/characters/README.md

```
@@ -0,0 +1,25 @@
+# Character Portraits
+
+Upload character portrait images here.
+
+## Required Files:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)